scalajslib/worker/0.6/src/ScalaJSWorkerImpl.scala

```diff
@@ -3,29 +3,33 @@ package scalajslib
 package worker
 
 import java.io.File
-import java.net.URLClassLoader
 
 import mill.api.Result
+import mill.scalajslib.api.{JsEnvConfig, ModuleKind}
 import org.scalajs.core.tools.io.IRFileCache.IRContainer
 import org.scalajs.core.tools.io._
 import org.scalajs.core.tools.jsdep.ResolvedJSDependency
 import org.scalajs.core.tools.linker.{ModuleInitializer, Semantics, StandardLinker, ModuleKind => ScalaJSModuleKind}
 import org.scalajs.core.tools.logging.ScalaConsoleLogger
 import org.scalajs.jsenv._
 import org.scalajs.testadapter.TestAdapter
-import mill.scalajslib.api.{JsEnvConfig, ModuleKind}
+
 class ScalaJSWorkerImpl extends mill.scalajslib.api.ScalaJSWorkerApi {
+    
+  var scalaJSModuleKind: ScalaJSModuleKind = null;
```

Comment:
@javaduke we should not have mutable variables if we can avoid it. If we need this field in `getFramework`, then we should pass it in

Reply:
Good point, fixed.